src/modules/Lan.rb

```diff
@@ -1033,23 +1033,18 @@ def configure_as_bridge!(ifcfg, bridge_name)
       NetworkInterfaces.Commit
     end
 
-    def propose_current_item!(config)
+    def proposable_as_bridge?(config)
       # first configure all connected unconfigured devices with dhcp (with default parameters)
       hwinfo = config.fetch("hwinfo", {})
-
       if hwinfo.fetch("link", false) == true
-        log.warn("item number #{LanItems.current} has link:false detected")
-
+        log.warn("Ifcfg #{config["ifcfg"]} has link:false detected")
```

Comment:
and what if the device is not configured?

Reply:
If it is not configured and it is bridgeable we will propose it. I could pass current instead of use config["ifcfg"] or we could set the `LanItems.current` before if we consider it is mandatory to have there the log, but probably just adding the current item into the log inside of the comparison should be enough